release/pipline/dags/istio_common_dag.py

```diff
@@ -185,10 +196,12 @@ def GenerateTestArgs(**kwargs):
       dag=common_dag)
 
   build_template = """
-    chmod +x /home/airflow/gcs/data/release/*
     {% set settings = task_instance.xcom_pull(task_ids='generate_workflow_args') %}
     {% set m_commit = task_instance.xcom_pull(task_ids='get_git_commit') %}
-    /home/airflow/gcs/data/release/start_gcb_build.sh -w -p {{ settings.PROJECT_ID \
+    gsutil cp gs://{{ settings.GCS_RELEASE_TOOLS_PATH }}/data/release/*.json .
+    gsutil cp gs://{{ settings.GCS_RELEASE_TOOLS_PATH }}/data/release/*.sh .
+    chmod +x *
```

Comment:
I'd feel more at ease with "chmod +x *.sh", but practically speaking it doesn't matter.  Is there any need to be concerned about multiple tasks running concurrently and thus each task could pull in different set of files and stop over each other?

Reply:
There is no need to be concerned about multiple tasks running concurrently.